showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITHttpAnnotation.java

```diff
@@ -0,0 +1,119 @@
+package com.google.showcase.v1beta1.it;
```

Comment:
We may be missing a copyright header for this file?

Hm I wonder why our checkstyle didn't fail.

Reply:
Added the copyright header. I'll need to check why it didn't run for the test files.

Reply:
Thanks @lqiu96!